client/src/components/Users.js

```diff
@@ -1,39 +1,86 @@
 import React from 'react';
 import axios from 'axios';
+import styled, {ThemeProvider} from 'styled-components';
 
-const ax = axios.create({
-  baseUrl: 'http://localhost:5000', // this isn't working for some reason
-});
+// axios instance with base url to simplify requests
+//const ax = axios.create({
+//baseURL: 'http://localhost:5000',
+//});
 
-class Users extends React.Component {
-  constructor() {
-    super();
-    this.state = {
-      users: [],
-    };
-  }
+// theme for styled components
+const theme = {
+  granite: '#6d6466',
+  gray: '#9f9f92',
+  lightGreen: '#d0fcb3',
+  darkGreen: '#9bc59d',
+  black: '#271f30',
+};
+
+const UserList = styled.div`
+  width: 60%;
+  border: 2px solid ${props => props.theme.black};
+  background-color: ${props => props.theme.darkGreen};
+  margin: 0 auto;
+  box-shadow: 4px 4px 10px ${props => props.theme.black};
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+`;
+
+const User = styled.div`
+  border: 2px solid ${props => props.theme.black};
+  width: 50%;
+  padding: 10px;
+  margin: 10px;
+  box-shadow: 2px 2px 10px ${props => props.theme.granite};
+  color: ${props => props.theme.granite};
+  transition: all 0.2s ease-out;
+  border-radius: 5px;
 
-  componentDidMount() {
-    ax.get('http://localhost:5000/users')
-      .then(res => {
-        this.setState({users: res.data});
-      })
-      .catch(err =>
-        console.log(`there was an error fetching the users: ${err}`),
-      );
+  &:hover {
+    color: ${props => props.theme.black};
+    font-weight: bold;
+    transform: scale(1.2, 1.5);
+    background-color: ${props => props.theme.gray};
+    border-radius: 200px;
   }
+`;
 
-  render() {
-    if (!this.state.users.length) return <h3>loading</h3>;
-    return (
-      <div>
+const Users = props => {
+  //constructor() {
+  //super();
+  //this.state = {
+  //users: [],
+  //};
+  //}
+
+  //componentDidMount() {
+  //ax.get('/users')
+  //.then(res => {
+  //this.setState({users: res.data});
+  //})
+  //.catch(err =>
+  //console.log(`there was an error fetching the users: ${err}`),
+  //);
+  //}
+
+  if (!props.users.length) return <h3>loading</h3>;
+  return (
+    <ThemeProvider theme={theme}>
```

Comment:
You're really good w/ styled-components, first case of using theme I can remember seeing. (It could be because I just became aware of it recently so now I know what it is, haha)